README.md

```diff
@@ -12,43 +12,73 @@
  License.
 -->
 
-# Google Cloud Platform: github-actions
+# `setup-gcloud` GitHub Action
 
-This repository contains a library of [Github Actions](https://github.com/actions) providing functionality for working with [Google Cloud Platform](http://cloud.google.com/).
+Configures the [Google Cloud SDK][sdk] in the GitHub Actions environment. The Google Cloud SDK includes both the [gcloud][gcloud] and
+[gsutil][gsutil] binaries.
 
-## Available Actions
+Find other Google Cloud GitHub Actions in [`google-github-actions`](https://github.com/google-github-actions).
```

Comment:
Since majority of our users may use setup-gcloud, can we surface some of the other actions here so they can make a decision of setup-gcloud vs native action.

Reply:
Do we like the actions listed in this way?

Reply:
Yup those look great and increased visibility!